### PR TITLE
FreeRDP: Update to 2.0.0-rc1

### PR DESCRIPTION
**Pull request by metalefty**
#### Description

Most of the local patches are no longer needed since upstream improved macOS compatibility.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.3 17D47
Xcode 8.3 8E162

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

I overlooked this:
> Please don't open a new Trac ticket if you are submitting a pull request.

so I created a new Trac ticket, sorry.

https://trac.macports.org/ticket/55829

### CODE REVIEW

**Comment by mf2k** [comment]
Can you please explain, and in the commit message, why you are removing the pulseaudio variant?

**Comment by metalefty** [comment]
Because FreeRDP on Mac no longer require PulseAudio to plackback remote audio locally. AFAIK PulseAudio is required only for that purpose in FreeRDP 1.1.0 port.

I'll add the explanation in commit message later.

**Comment by metalefty** [comment]
@mf2k I've updated the commit message.

File: net/FreeRDP/Portfile [hunk 1]
```diff
@@ -4,13 +4,8 @@ PortSystem          1.0
 PortGroup           github  1.0
 PortGroup           cmake   1.0
 
-epoch               2
```
[the remark this answers is not in the document]
Comment (metalefty) [reply net/FreeRDP/Portfile]: Sorry, I unintendedly removed it. Should I restore it by my self?

[the remark this answers is not in the document]
Comment (pmetzger) [reply net/FreeRDP/Portfile]: Yah. Please carefully check all your changes, fix it, and push a new version. (Squash all the commits in the new version, we prefer to pull a single commit onto the tree.)

**Comment by pmetzger** [comment]
Travis timed out, but this looks good. Merged. Thank you for your contribution to MacPorts!

**Pull request merged**